### PR TITLE
Attempt at fixing the flaky scheduled triggers test

### PR DESCRIPTION
**Pull request by codingkarthik**
The `test_check_fired_webhook_event` test is flaky in the `v1.3.0` branch. This PR attempts to solve that. 

### Changelog

- [x] `CHANGELOG.md` is updated with user-facing content relevant to this PR. If no changelog is required, then add the `no-changelog-required` label.

### Affected components

- [ ] Server
- [ ] Console
- [ ] CLI
- [ ] Docs
- [ ] Community Content
- [ ] Build System
- [x] Tests
- [ ] Other (list it)

#### Catalog upgrade

Does this PR change Hasura Catalog version?
- [ ] No
- [ ] Yes
  - [ ] Updated docs with SQL for downgrading the catalog 

#### Metadata

Does this PR add a new Metadata feature?
- [ ] No
- [ ] Yes
  - Does `run_sql` auto manages the new metadata through schema diffing?
    - [ ] Yes
    - [ ] Not required
  - Does `run_sql` auto manages the definitions of metadata on renaming?
    - [ ] Yes
    - [ ] Not required
  - Does `export_metadata`/`replace_metadata` supports the new metadata added?
    - [ ] Yes
    - [ ] Not required

#### GraphQL
- [ ] No new GraphQL schema is generated
- [ ] New GraphQL schema is being generated:
   - [ ] New types and typenames are correlated
   
   

#### Breaking changes

- [ ] No Breaking changes
- [ ] There are breaking changes:

  1. Metadata API

     Existing `query` types:
     - [ ] Modify `args` payload which is not backward compatible
     - [ ] Behavioural change of the API
     - [ ] Change in response `JSON` schema
     - [ ] Change in error code
     

  2. GraphQL API

     Schema Generation:
     
     
     - [ ] Change in any `NamedType`
     - [ ] Change in table field names
     

     Schema Resolve:-
     
     - [ ] Change in treatment of `null` value for any input fields 
     

  3. Logging

     - [ ] Log `JSON` schema has changed
     - [ ] Log `type` names have changed

### CODE REVIEW

**Comment by codingkarthik** [comment]
I have run the tests for 3 time now, now the flaky test seems to be stable.

File: server/tests-py/test_scheduled_triggers.py [hunk 1]
```diff
@@ -78,7 +78,10 @@ def test_create_trigger_with_error_returning_webhook(self,hge_ctx):
         assert st == 200, resp
 
     def test_check_fired_webhook_event(self,hge_ctx,scheduled_triggers_evts_webhook):
-        event = scheduled_triggers_evts_webhook.get_event(65)
+        # Ideally, the timeout should be 60 seconds, within which a scheduled event
+        # should be delivered, but this test is flaky while it's run in CI. So,
+        # increasing the timeout to 120 seconds.
+        event = scheduled_triggers_evts_webhook.get_event(120)
```
Comment (tirumaraiselvan) [inline server/tests-py/test_scheduled_triggers.py:84]: Isn't 120s kind of an overkill. Can we try 70?

Comment (codingkarthik) [reply server/tests-py/test_scheduled_triggers.py]: The 120s here is the max timeout to get the event , but I agree that 120 seems to be an overkill. I'll experiment with lower values starting from 70.

Comment (tirumaraiselvan) [reply server/tests-py/test_scheduled_triggers.py]: Do you know why 65s sometimes doesn't work?

Comment (codingkarthik) [reply server/tests-py/test_scheduled_triggers.py]: We run the tests parallelly in some of the configurations, which may cause the flakiness of the test. I have added a decorator to the tests to run the tests sequentially with timeout as 65. But still not 100% sure if this was the problem.

**Comment by codingkarthik** [comment]
Seems like this issue no longer exists, I will open this PR if the test flakes again

**Comment by Stefmore02** [comment]
Closing this PR as is not relevant anymore.

**Pull request closed without merging**